### PR TITLE
Skip latest

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

File: test/test.py [hunk 1]
```diff
@@ -267,8 +275,10 @@ def testScores(self):
             self.assertEqual(3.0, res.docs[0].score)
 
             self.assertEqual('doc1', res.docs[1].id)
-            # todo: enable once new RS version is tagged
-            #self.assertEqual(0.2, res.docs[1].score)
+            
+            if check_version_latest(r) is False:
```
Comment (rafie) [inline test/test.py:277]: `if not check_version_latest(r):`

File: test/test.py [hunk 2]
```diff
@@ -267,8 +275,10 @@ def testScores(self):
...
+                self.assertEqual(0.2, res.docs[1].score)
+                print 'something'
```
Comment (rafie) [inline test/test.py]: what.
:-)

File: test/test.py [hunk 3]
```diff
@@ -47,6 +47,14 @@ def check_version_2(env):
             return True
         return False
 
+def check_version_latest(env):
+    res = env.execute_command('info modules')
+    latest = res.find('ver=999999')
+    print latest
+    if latest is not -1:
+        return True
+    return False
```
Comment (rafie) [inline test/test.py:55]: `return latest is not -1`

**Comment by ashtul** [comment]
replaced by #106

**Pull request closed without merging**